core/src/main/java/io/grpc/internal/SerializingExecutor.java

```diff
@@ -55,6 +58,60 @@ private static AtomicHelper getAtomicHelper() {
     return helper;
   }
 
+  private static final Class<? extends Executor> threadlessExecutorClass;
+  private static final Class<? extends Executor> guavaSeqExecutorClass;
+
+  static {
+    boolean wrapSerialized = Boolean.getBoolean("grpc.wrapSerialized");
```

Comment:
This is here to make the benchmark comparisons easier, can be tidied up if this is accepted.